index.js

```diff
@@ -25,33 +25,31 @@ function search(buf) {
 	}
 
 	return false;
-}
-
-exports.buffer = function (buf) {
-	return search(buf);
 };
 
-exports.stream = function (cb) {
-	var prevLastByte = new Buffer(1);
-	var searchDone = false;
+exports.buffer = search;
+
+exports.stream = cb => {
+	let prevLastByte = new Buffer(1);
```

Comment:
Is it possible to somehow do this differently or is it ok for this function to use a callback?